docs/workload-development-kit/faq.md

```diff
@@ -17,7 +17,8 @@ This article contains frequently asked questions relating to the Microsoft Workl
 
 ### I have a question that isn't answered in this FAQ, where can I get help?
 
-Reach out to [Fabric workload dev support](mailto:ILDCWLSupport@microsoft.com).
+Please search and post to [Microsoft-Fabric-workload-development-sample/discussions](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/discussions) 
```

Comment:
Remove 'please' - we don't use that in technical writing. Also, remove search - it's not needed.

Go with something like this:

Post your question at xxx or yyy so that it can be addressed by the Fabric team.